### ISSUE TITLE
Main font option not taken into account

### ISSUE THREAD

**Issue by Brifien** [doublecmd/doublecmd#161]
In Configuration - Options - Fonts, the "Main font" is not taken into account when changed.

Operating system: Windows
Revision: 1.0.0 alpha 9890
(doublecmd-1.0.0.r9890.x86_64-win64)

**Comment by alexx2000** [comment]
Works fine.

**Comment by Brifien** [comment]
Thanks for your response.

With my "old" profile, it was not working.

![2021-08-24_083607](https://user-images.githubusercontent.com/89442281/130573914-664494e5-e6c4-4951-82aa-7b9dfb849317.png)
![2021-08-24_091415](https://user-images.githubusercontent.com/89442281/130573934-f89b326b-c430-49c0-8776-3376324fe160.png)

But I tried with a new profile (I deleted doublecmd folder in \AppData\Roaming) and it is now working.

Thanks
